src/libs/actions/PushNotification.js

```diff
@@ -0,0 +1,44 @@
+import Onyx from 'react-native-onyx';
+import CONST from '../../CONST';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as API from '../API';
+import * as Device from './Device';
+
+let isUserOptedInToPushNotifications = false;
+Onyx.connect({
+    key: ONYXKEYS.PUSH_NOTIFICATIONS_ENABLED,
+    callback: val => isUserOptedInToPushNotifications = val,
+});
+
+/**
+ * Record that user opted-in or opted-out of push notifications on the current device.
+ * NOTE: This is purely for record-keeping purposes, and does not affect whether our server will attempt to send notifications to this user.
```

Comment:
Ohhh, I think this is what I misunderstood.

Reply:
Yeah, this is still true for now. However, we _could_ use this to control how and when and where notifications are sent from the server since this PR is quite robust in tracking opt-in. We just aren't currently. So I should probably update this comment so that we don't leave assumptions about a different codebase in a comment in this codebase.

Reply:
Nice. We could probably add this task to the [Notifications phase 2 tracking issue](https://github.com/Expensify/Expensify/issues/255265) if you liked.